Is this the output you are seeing?:

```
E   -                'properties': {'amenity': 'parking',
E   ?                                ^^^^ ^^    ^ ^^  ^
E
E   +                'properties': {'@id': 'relation/5576903',
E   ?                                ^ ^    ^^^ ^ + ^^^^^^^^
E
E   +                               'amenity': 'parking',
E                                   'fee': 'yes',
E   -                               'id': 'relation/5576903',
```

If you are running the test on branch issue_3, it fails, on master with #7 merged before, it should pass.

Can you confirm or provide the error message you get?

--- a/Changelog.md
+++ b/Changelog.md
@@ -2,7 +2,12 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
-and this project adheres to [Semantic Versioning](http://semver.org/).
+and this project will later on adhere to [Semantic Versioning](http://semver.org/). 
+
+Note: A change of a minor/patch version still may result in changed GeoJSON export results!  
 
 ## [0.0.1]
     * Project creation
+## [0.0.2]
+    * Prefix osm "id" property by an "at"-sign (fixes #5)
+    * Create GeometryCollection for relations with mixed member types (fixes #3)

--- a/setup.py
+++ b/setup.py
@@ -2,7 +2,7 @@ from setuptools import setup, find_packages
 
 setup(
     name='osmtogeojson',
-    version='0.0.1',
+    version='0.0.2',
     packages=find_packages(exclude=["tests.*", "tests"]),
     author="Tommy Carpenter",
     author_email="",

